functions.Makefile

```diff
@@ -346,6 +346,9 @@ define __ecs_target_template
 $(1)-build:
 	$(call build_image,$(1),$(2))
 
+$(1)-test:
+	$(call test_lambda,$(STACK_ROOT)/$(1))
```

Comment:
Should this be test_ecs or test_image or something like that?

Reply:
yup, it definitely should :)